Quadriga/src/test/java/edu/asu/spring/quadriga/web/config/impl/UserHelperTest.java

```diff
@@ -0,0 +1,97 @@
+package edu.asu.spring.quadriga.web.config.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.social.connect.Connection;
+import org.springframework.social.connect.ConnectionKey;
+import org.springframework.social.connect.UserProfile;
+
+import edu.asu.spring.quadriga.domain.IUser;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.impl.UserManager;
+
+/**
+ * This class consists of unit tests for UserHelper.
+ * 
+ * @author Chiraag Subramanian
+ *
+ */
+public class UserHelperTest {
+    
+    private final Logger logger = LoggerFactory.getLogger(UserHelperTest.class);
+    
+    @Mock
+    private UserManager userManager = Mockito.mock(UserManager.class);
+    
+    @InjectMocks
+    private UserHelper userHelper;
```

Comment:
let's follow this style guide: https://diging.atlassian.net/wiki/spaces/DIGING/pages/49577990/Unit+Tests+-+Java